README-linux.md

```diff
@@ -0,0 +1,242 @@
+# Quilibrium - Dawn
+
+Quilibrium is a decentralized alternative to platform as a service providers.
+This release, mirrored to GitHub, is the Dawn release, which contains the
+initial application, the MPC Powers-of-Tau Ceremony. Documentation for the
+underlying technology can be found at https://www.quilibrium.com/
+
+## Install Requirements
+
+    wget https://:go.dev/dl/go1.20.14.linux-amd64.tar.gz
+    sudo tar -xvf go1.20.14.linux-amd64.tar.gz
+    sudo mv go /usr/local
+    sudo rm go1.20.14.linux-amd64.tar.gz
+    sudo nano ~/.bashrc
+
+At the end of the file, add these lines and save the file.
+
+    GOROOT=/usr/local/go
+    GOPATH=$HOME/go
+    PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+
+On command line, run 
+    ~/.bashrc
+
+Check GO Version
+    go version
+
+It must show "go version go.1.20.13 linux/amd64"
```

Comment:
This should be 1.20.14 given the above instructions, but we should probably make sure these instructions direct people to the latest of 1.20